scripts/relayer/init_chain_2.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
```

Comment:
I'm guessing this is used to start another node in parallel to a running node. Can you put a small comment here to the script usage for future reference?

Reply:
done